spec/index.bs

```diff
@@ -25,7 +25,31 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
 Introduction {#intro}
 =====================
 
-This specification rocks.
+*This section is non-normative.*
+
+[Graphics Processing Units](https://en.wikipedia.org/wiki/Graphics_processing_unit), or GPUs for short,
+have been essential in enabling rich rendering and computational applications in personal computing.
+WebGPU is an API that exposes the capabilities of GPU hardware for the Web.
+The API is designed from the ground up to efficiently map to [Vulkan](https://www.khronos.org/vulkan/),
+[Direct3D 12](https://docs.microsoft.com/en-us/windows/win32/direct3d12/what-is-directx-12-),
+and [Metal](https://developer.apple.com/metal/) native GPU APIs.
+WebGPU is not related to [WebGL](https://www.khronos.org/webgl/) and does not explicitly target OpenGL ES.
+
+WebGPU sees physical GPU hardware as {{GPUAdapter}}s. It provides a connection to an adapter via
+{{GPUDevice}}, which manages resources, and the device's {{GPUQueue}}s, which execute commands.
+{{GPUDevice}} may have its own memory with high-speed access to the processing units.
```

Comment:
This is written with the idea that one can see the same GPU as different adapters via different APIs (e.g. on Windows it can be accessed via D3D12 or Vulkan). So we end up with multiple `GPUAdapter` instances, in general, per physical GPU.